contracts/sol6/KyberNetwork.sol

```diff
@@ -814,9 +814,8 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
         tradeData.networkFeeWei =
             (((tradeData.tradeWei * tradeData.networkFeeBps) / BPS) * tradeData.feeAccountedBps) /
             BPS;
-        require(
-            tradeData.tradeWei >= (tradeData.networkFeeWei + tradeData.platformFeeWei),
-            "fees exceed trade"
+        assert(
```

Comment:
can this be 1 line? or linter complains?

Reply:
good idea
done